.buildkite/pipeline.yml

```diff
@@ -3,18 +3,18 @@ steps:
     agents:
       builder: "dind"
     label: "helm template check for sanity"
-    if: build.message !~ /feat\(helm-dashboard\):/i
+    if: build.message !~ /feat\(OSS.+\):/i
```

Comment:
@undera You'll need to update https://github.com/komodorio/helm-dashboard/blob/main/.github/workflows/publish-chart.yaml#L43 this line as well to include `OSS` in the commit message of helm-dashboard to not break it's own CI

Reply:
This is exactly what I've prepared, and I'm waiting for this PR to be merged, to followup with change in helm-dashboard repo. Do you want me to do the change before this PR merges?

Reply:
@MickaelAlliel anyway, here's corresponding PR: https://github.com/komodorio/helm-dashboard/pull/369

Reply:
Didn't think too much about the order I'll be honest.
But since you've already merged the helm-dashboard one, let's merge this one now :)